rasa/test.py

```diff
@@ -151,10 +152,11 @@ def test_core(
         additional_arguments, test, ["stories", "agent"]
     )
 
-    rasa.utils.common.run_in_loop(test(stories, _agent, out_directory=output, **kwargs))
+    # rasa.utils.common.run_in_loop(test(stories, _agent, out_directory=output, **kwargs))
```

Comment:
Good catch! The error is raised as this line (currently commented out) creates a second loop although the one which you create in `rasa.cli.test` is already running. So having `test_core` being `async` is actually a very good idea 👍🏻